Added gift

ismaro3 opened the pull request.

fjlopez commented:
Two quick fixes and you have the :gift:. First, be consistent. In [Server.java](https://github.com/ismaro3/Laboratory-3-Restful-Web-Services/blob/master/src/main/java/rest/addressbook/Server.java) the server uses 8080 but data uses 8282. And second, you should provide information in 404 errors. For example, if I navigate to ```http://localhost:8282/contacts/person/3``` you should provide something useful.  Google can provide [some useful ideas](https://google.github.io/styleguide/jsoncstyleguide.xml)

ismaro3 commented:
Okey, I fixed consistency in Server.Java and added useful information in case of 404 errors:
A JSON-API error message is returned, indicating whether the 404 is caused because an operation with a non-existent person has been requested, or because it has been made with an incorrect path (e.g: /contacts/nonexistent).
